tests/console/slp.pm

```diff
@@ -27,18 +27,32 @@ sub run {
     my ($self) = @_;
     select_console 'root-console';
 
-    zypper_call 'in openslp openslp-server';
+    # Workaround: slpd requires its own user
+    # %pre statement in spec file of openslp-server creates dependency to daemon group provided by *system-user-daemon*
+    # /usr/sbin/useradd -r -g daemon -d /var/lib/empty -s /sbin/nologin -c "openslp daemon" openslp 2>/dev/null || :
+    # up to this point, JeOS doesn't require *system-user-daemon*
+    if (script_run 'getent group daemon') {
+        zypper_call 'in system-user-daemon';
+        assert_script_run 'useradd -d /var/lib/empty/ -s /sbin/nologin openslp';
+        record_soft_failure 'bsc#1165050 openslp-server does not create openslp user when system-user-daemon is not installed';
+    }
+
+    # Let's install slpd
+    zypper_call 'in openslp-server';
```

Comment:
Wasn't there a product issue that the server doesn't work unless the client is also installed?

Reply:
Indeed! Nice catch